sem_covid/entrypoints/etl_dags/finreg.py

```diff
@@ -0,0 +1,282 @@
+        ev:871
+        ev:1497
+        ev:5090
+        ev:57
+        ev:924
+        ev:3240
+        ev:3942
+        ev:1130
+        ev:1326
+        ev:5156
+        ev:5645
+        ev:6343
+        ev:1971
+        ev:2149
+        ev:2831
+        ev:4115
+        ev:1459
+        ev:560
+        ev:1972
+        ev:5964
+        ev:4250
+        ev:5465
+        ev:5598
+        ev:170
+        ev:2839
+        ev:4491
+        ev:6362
+        ev:728
+        ev:1787
+        ev:834
+        ev:3148
+        ev:4855
+        ev:2219
+        ev:5566
+        ev:c_406ad4cc
+        ev:c_dcc650ef
+        ev:c_3e6af2e7
+        ev:c_8f89faac
+        ev:c_96124aaf
+        ev:c_b499ede2
+        ev:c_a18525ab
+        ev:c_e749c083
+        ev:c_dd52c1e9
+        ev:c_dcf3f7c0
+        ev:c_a3b85311
+        ev:1000
+        ev:1460
+        ev:2497
+        ev:2900
+        ev:4838
+        ev:2504
+        ev:1448
+        ev:4190
+        ev:6751
+        ev:165
+        ev:2607
+        ev:1492
+        ev:5389
+        ev:5218
+        ev:3543
+        ev:3233
+        ev:1801
+        ev:1815
+        ev:4703
+        ev:2162
+        ev:55
+        ev:3119
+        ev:1491
+        ev:115
+        ev:524
+        ev:1490
+        ev:822
+        ev:4406
+        ev:3235
+        ev:c_14d71455
+        ev:c_7f2d2214
+        ev:c_d08207d1
+        ev:1313
+        ev:4279
+    }
+    VALUES ?resource_type {
+        res_type:REG
+        fd_030:REGL
+        res_type:DIR
+        res_type:REG_IMPL
+        res_type:REG_DEL
+        fd_030:DIRECTIVE
+        fd_030:REGIMP
+        fd_030:REGDEL
+        res_type:DIR_DEL
+        res_type:DIR_IMPL
+        res_type:REG_FINANC
+        fd_030:DIRDEL
+        fd_030:DIRIMP
+        fd_030:DIR
+        fd_030:REG
+    }
+
+    ?work cdm:work_is_about_concept_eurovoc ?eurovoc_concept ;
+        cdm:work_has_resource-type|cdm:resource_legal_has_type_act_concept_type_act ?resource_type .
+    OPTIONAL {
+        ?exp cdm:expression_belongs_to_work ?work ;
+            cdm:expression_uses_language ?expr_lang ;
+            cdm:expression_title ?_title .
+    }
+}
+GROUP BY ?work'''
+
+
+def make_request(query):
+    wrapper = SPARQLWrapper(config.EU_CELLAR_SPARQL_URL)
+    wrapper.setQuery(query)
+    wrapper.setReturnFormat(JSON)
+
+    return wrapper.query().convert()
+
+
+def get_single_item(query, json_file_name):
+    minio = StoreRegistry.minio_object_store(config.EU_FINREG_CELLAR_BUCKET_NAME)
+    response = make_request(query)['results']['bindings']
+    uploaded_bytes = minio.put_object(json_file_name, json.dumps(response).encode('utf-8'))
+    logger.info(f'Save query result to {json_file_name}')
+    logger.info('Uploaded ' +
+                str(uploaded_bytes) +
+                ' bytes to bucket [' +
+                config.EU_FINREG_CELLAR_BUCKET_NAME + '] at ' +
+                config.MINIO_URL)
+    return response
+
+
+def download_and_split_callable():
```

Comment:
not sure we need the "_callable" suffix

Reply:
i saw this practice in several articles. should i remove it ?